packages/xcb_proto.rb

```diff
@@ -0,0 +1,20 @@
+require 'package'
+
+class Xcb_proto < Package
+  description 'The protocols for the X window system provide extended functionality for communication between a X client and the server.'
+  homepage 'https://x.org'
+  version '1.12'
+  source_url 'https://www.x.org/archive/individual/xcb/xcb-proto-1.12.tar.gz'
+  source_sha256 'cfa49e65dd390233d560ce4476575e4b76e505a0e0bacdfb5ba6f8d0af53fd59'
+
+  depends_on 'python'
```

Comment:
since you are using x86_64.  it doesnt on arm.  please use either python3 or python27 as one of dependencies.